CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2021-09-30
+
+This release is rolling out to all regions - estimated time for completed roll out is 2021-10-07 for public cloud and 2021-10-14 for sovereign clouds.
+
+### Release Notes
+
+* Preview Features
+  * Enable multiple service account issuers for clusters using version >= 1.22, also fixing CNCF Validation issues.
+* Behavioral Changes
+  * When turning Cluster Auto Scaler off, you can now specify the requested agent pool node number.
```

Comment:
On the same API call, PUT/request?

Reply:
@marwanad can you comment?

Reply:
Tong made that change but yes that is my understanding.

Reply:
it was rhetorical, the question was to should we add that clarification?